libs/langchain/tests/unit_tests/schema/runnable/test_runnable.py

```diff
@@ -368,6 +369,66 @@ async def test_prompt() -> None:
         part async for part in prompt.astream({"question": "What is your name?"})
     ] == [expected]
 
+    stream_log = [
```

Comment:
Are there any other tests we could add? It feels a bit light on test coverage given how important this feature is